### PR TITLE
feat(#33): implement model input

### PR DESCRIPTION
**Pull request by Jshen123**
**Description:**
allow user to input custom model name

image

### CODE REVIEW

File: src/components/SettingsDialog.tsx [hunk 1]
```diff
@@ -56,11 +60,23 @@ export default function SettingsDialog({
             link.
           </a>
         </p>
+        <Input
```
Comment (awtkns) [inline src/components/SettingsDialog.tsx:63]: Ideally I think this should be a drop down. But we can make that improvement later.

File: src/components/SettingsDialog.tsx [hunk 2]
```diff
@@ -56,11 +60,23 @@ export default function SettingsDialog({
...
+          left={
+            <>
+              <FaMicrochip />
+              <span className="ml-2">Model:</span>
+            </>
+          }
+          placeholder={""}
```
Comment (awtkns) [inline src/components/SettingsDialog.tsx:70]: Might be good to have a place holder here. A default value would be even better 😊

File: src/components/SettingsDialog.tsx (hunk 1 above)
Comment (asim-shrestha) [reply src/components/SettingsDialog.tsx:63]: Told him drop down could wait for a diff push

File: src/components/SettingsDialog.tsx (hunk 2 above)
Comment (asim-shrestha) [reply src/components/SettingsDialog.tsx:70]: lets do a placeholder of gpt-3.5-turbo

**Review by asim-shrestha — APPROVED** [review]

**Pull request merged**